Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -170,13 +207,16 @@ $.connection.hub.start().done(function () {
 
         this.editItem = function () {
             var that = this;
-            viewModel.editText("Fetching changes...");
+            // Blank out the editor before fetching new content
+            viewModel.editText(null);
+            statusbarObj.FetchingChanges();
             viewModel.fileEdit(this);
             if(this.mime == "text/xml")
```

Comment:
`if(this.mime == "text/xml")` is not in my commit. Should I make the change anyway?